<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dolphin</string>
  </property>
  <widget class="QStackedWidget" name="centralWidget">
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionAddFolder"/>
    <addaction name="separator"/>
    <addaction name="actionRefresh"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEmulation">
    <property name="title">
     <string>Emulation</string>
    </property>
    <addaction name="actionPlay"/>
    <addaction name="actionPause"/>
    <addaction name="actionStop"/>
    <addaction name="separator"/>
    <addaction name="actionFullscreen"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionConfigure"/>
    <addaction name="separator"/>
    <addaction name="actionGraphics"/>
    <addaction name="actionSound"/>
    <addaction name="actionWiimote"/>
    <addaction name="actionGamecube"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionShowToolbar"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionWebsite"/>
    <addaction name="actionDocumentation"/>
    <addaction name="actionDevelopment"/>
    <addaction name="separator"/>
    <addaction name="actionReportIssue"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEmulation"/>
   <addaction name="menuOptions"/>
   <addaction name="menuTools"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>Main toolbar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionRefresh"/>
   <addaction name="actionAddFolder"/>
   <addaction name="separator"/>
   <addaction name="actionPlay"/>
   <addaction name="actionStop"/>
   <addaction name="separator"/>
   <addaction name="actionConfigure"/>
   <addaction name="actionGraphics"/>
   <addaction name="actionSound"/>
   <addaction name="actionGamecube"/>
   <addaction name="actionWiimote"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionAddFolder">
   <property name="text">
    <string>Browse</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="text">
    <string>Refresh</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="text">
    <string>Play</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionFullscreen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Fullscreen</string>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="text">
    <string>Configure</string>
   </property>
  </action>
  <action name="actionGraphics">
   <property name="text">
    <string>Graphics</string>
   </property>
  </action>
  <action name="actionSound">
   <property name="text">
    <string>Sound</string>
   </property>
  </action>
  <action name="actionWiimote">
   <property name="text">
    <string>Wiimote</string>
   </property>
  </action>
  <action name="actionGamecube">
   <property name="text">
    <string>Gamecube</string>
   </property>
  </action>
  <action name="actionShowToolbar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show toolbar</string>
   </property>
  </action>
  <action name="actionWebsite">
   <property name="text">
    <string>Website</string>
   </property>
  </action>
  <action name="actionDocumentation">
   <property name="text">
    <string>Documentation</string>
   </property>
  </action>
  <action name="actionDevelopment">
   <property name="text">
    <string>Development</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About...</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="text">
    <string>Pause</string>
   </property>
  </action>
  <action name="actionReportIssue">
   <property name="text">
    <string>Report issue...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionShowToolbar</sender>
   <signal>triggered(bool)</signal>
   <receiver>toolBar</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>32</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
